ivi-input-modules/ivi-input-controller/src/ivi-input-controller.c

```diff
@@ -561,24 +561,21 @@ input_ctrl_ptr_leave_west_focus(struct seat_ctx *ctx_seat,
 
 static void
 input_ctrl_ptr_set_west_focus(struct seat_ctx *ctx_seat,
-        struct weston_pointer *pointer, struct weston_view *w_view,
-        int rel_x, int rel_y)
+        struct weston_pointer *pointer, struct weston_view *view)
```

Comment:
Why did you change the name of the parameter from w_view to view in a bugfix patch ? Please do not mix bug-fixing changes and  style changes in one patch.

Reply:
ok. I will revert the name changes.